packages/o-spreadsheet-engine/src/plugins/core/cell.ts

```diff
@@ -115,23 +104,10 @@ export class CellPlugin extends CorePlugin<CoreState> implements CoreState {
 
   handle(cmd: CoreCommand) {
     switch (cmd.type) {
-      case "SET_FORMATTING":
-        if ("format" in cmd && cmd.format !== undefined) {
-          this.setFormatter(cmd.sheetId, cmd.target, cmd.format);
-        }
-        break;
-      case "CLEAR_FORMATTING":
-        this.clearFormatting(cmd.sheetId, cmd.target);
-        break;
-      case "ADD_COLUMNS_ROWS":
-        if (cmd.dimension === "COL") {
-          this.handleAddColumnsRows(cmd, this.copyColumnFormat.bind(this));
-        } else {
-          this.handleAddColumnsRows(cmd, this.copyRowFormat.bind(this));
-        }
-        break;
       case "UPDATE_CELL":
-        this.updateCell(cmd.sheetId, cmd.col, cmd.row, cmd);
+        if ("content" in cmd || "formula" in cmd) {
```

Comment:
`"formula"` ? 🤔 

It looks like a relic from the past and could be removed completely